content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-token.md

```diff
@@ -0,0 +1,27 @@
+---
+title: kubeadm token
+content_type: conceito
+weight: 70
+---
+<!-- overview -->
+
+Os Bootstrap tokens são usados para estabelecer uma relação de confiança bidirecional entre um nó que se junta ao cluster e um nó do plano de controle, conforme descrito na [autenticação com tokens de inicialização](/docs/reference/access-authn-authz/bootstrap-tokens/).
+
+O `kubeadm init` cria um token inicial com um TTL de 24 horas. Os comandos a seguir permitem que você gerencie esse token e também crie e gerencie os novos.
+
+<!-- body -->
+## Criar um token kubeadm {#cmd-token-create}
+{{< include "generated/kubeadm_token_create.md" >}}
```

Comment:
You have to copy `content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_token_create.md` to `content/pt-br/`, it caused build failed.

Reply:
Thank you. /unhold